assets/js/components/Dashboard/AlertNotFoundPage.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Button } from "react-bootstrap";
+import { Search } from "react-bootstrap-icons";
+import { useNavigate } from "react-router-dom";
+
+const AlertNotFoundPage = (props: { validAlertId: string | undefined }) => {
+  const navigate = useNavigate();
+
+  return (
+    <div className="alert-not-found alert-not-found_page">
+      <Search size={24} className="alert-not-found_icon" />
+      <div>
+        <div className="alert-not-found_title">Alert not found</div>
+        <p className="alert-not-found_detail">
+          {props.validAlertId
+            ? `We couldn’t find the alert you’re looking for in Screenplay. This alert \
+            (ID #${props.validAlertId}) is open, but it won’t show up on any screens due to \
+            its Service Type, Effect Type, or Effect Period.`
+            : `We couldn’t find the alert you’re looking for in Screenplay. This could be \
+            because the alert has been closed, or because an alert with that ID \
+            (#${
+              window.location.pathname.split("/").slice(-1)[0]
```

Comment:
react-router provides a hook that can be used here instead of looking at `window`: `useLocation`. The basic string splitting logic would still be the same so you wouldn't gain much, so just a suggestion.